core/modules/czar/python/app.py

```diff
@@ -292,7 +289,7 @@ def __init__(self, query, context, setSessionId, resultName=""):
         @param context a user context object containing conditions and a user query factory
         @param setSessionId - unary function. a callback so this object can provide
                           a handle (sessionId) for the caller to access query
-                          messages.
+                           messages.
         @param resultName name of result table for query results."""
```

Comment:
why this extra space that is breaking alignment? I'd put "messages." in previous line.